index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="es-MX">
+<html lang="es-ES">

```

Comment:
Why is this important? It seems that is not necessary for our current user base.